content/docs/how-we-work/security.md

```diff
@@ -14,15 +14,13 @@ For example, an application relies on database credentials to connect to the dat
 
 ## Server and Website Security
 
-In particular, we practice server and site security by:
+We prioritize server and site security in our operations. This involves the use of sanitized databases, which entails the modification or removal of sensitive information to safeguard data privacy and security. Additionally, we routinely scrub unneeded data from our development environments, ensuring that any data no longer necessary for the project is removed and that the remaining data is securely stored.
```

Comment:
Prioritize over what? If it's just a word, then I would suggest "We are mindful of server and site security"

Reply:
Changes made accordingly.